### DISCUSSION TITLE
Markdown filename for obsidian

### DISCUSSION THREAD

**Discussion by davidoskky** [windingwind/zotero-better-notes#564] [Note Templates]
This template allows you to export the notes with the default filename used by the obsidian plugin [citation](https://github.com/hans/obsidian-citation-plugin).

Which is: @{title}{year}

```
@${(noteItem.parentItem && noteItem.parentItem.getField("title")).replace(/[/\\?%*:|"<>]/g, "_") + (noteItem.parentItem && noteItem.parentItem.getField("year"))|| noteItem.key}.md
```

**Comment by Kmfernan5** [comment]
How do you use it?

**Comment by elfomze** [comment]
When I try to use this as a fresh template or by replacing the contents of the default [ExportMDFileName] template, I get the error `ReferenceError: noteItem is not defined`. This happens no matter what item I have selected when trying to use the template. Using the [Item] tag reproduces the same issue.

**Reply by windingwind** [reply]
please provide a screenshot of the note template editor with this template selected.
